CZ/pyranda_rayleigh_taylor/simulation_pgen.py

```diff
@@ -2,19 +2,19 @@
 from trata.sampler import LatinHyperCubeSampler as LHS
 from maestrowf.datastructures.core import ParameterGenerator
 
-# Settings for the Latinhypercube sampler
-Nruns = 15
-test_box = [[0.3, 0.65], [0.85, 1.15]]
-seed = 7
+def get_custom_generator(env, **kwargs):
+    # Settings for the Latinhypercube sampler
+    Nruns = int(kwargs.get("NRUNS", env.find("NRUNS").value))
+    test_box = [[0.3, 0.65], [0.85, 1.15]]
+    seed = 7
```

Comment:
is this seed separate from the SEED variable we'll be using in the spec file?

Reply:
Yes it's for the latin hypercube